Source/Core/Core/PowerPC/Interpreter/Interpreter_SystemRegisters.cpp

```diff
@@ -405,7 +405,7 @@ void Interpreter::mtspr(UGeckoInstruction inst)
   case SPR_DEC:
     if (!(old_value >> 31) && (rGPR[inst.RD] >> 31))  // top bit from 0 to 1
     {
-      PanicAlert("Interesting - Software triggered Decrementer exception");
+      INFO_LOG(POWERPC, "Interesting - Software triggered Decrementer exception");
```

Comment:
Any reason to leave the "Interresting" marker there?
It might've been back in the day that someone wanted to verify behavior and was desperately looking for something to trigger this (hence the `PanicAlert`), but I don't think it still has any value today.

Reply:
Nope, no reason. I've amended the commit to remove it.